sources/packages/api/src/database/entities/user.model.ts

```diff
@@ -32,4 +39,19 @@ export class User extends BaseModel {
     name: "is_active",
   })
   isActive: boolean;
+
+  /**
+   * SIN Validation ids related to the user.
+   */
+  @RelationId((user: User) => user.sinValidations)
+  sinValidationsIds: number[];
+  /**
+   * SIN Validation related to the user.
+   */
+  @OneToMany(() => SINValidation, (sinValidation) => sinValidation.user, {
+    eager: false,
+    cascade: true,
+    onDelete: "CASCADE",
```

Comment:
default is nullable, 
![image](https://user-images.githubusercontent.com/77353155/157529810-a24855fb-e159-4411-948a-63043732be9f.png)

Reply:
Its not a column, and just a relation between the user and the sinValidation table. So i think it can be null.

Reply:
we can add nullable to relation, in this PR, you did it 
![image](https://user-images.githubusercontent.com/77353155/157558618-ebbcbdae-8e17-42c6-9258-f2d7094d2e35.png)

Reply:
That's an actual column in the db, we have added in the sin_validations table. But this is just a relation, there is a difference. Even if we are not defaulting , the value will be null, I can add it to restrict further comments.

Reply:
removed this as part of new change